index.d.ts

```diff
@@ -8,6 +8,13 @@ declare namespace elementReady {
 		@default document
 		*/
 		readonly target?: Element | Document;
+
+		/**
+		Whether to stop when document has finished loading.
```

Comment:
The description should be the same as in the readme. This makes it easier to keep them in sync.